### ISSUE TITLE
[request] we need new levels :(

### ISSUE THREAD

**Issue by RIOT-Panko** [HackerPoet/NonEuclidean#30]
Does anyone make levels ?

**Comment by IPlayMinecraftYT** [comment]
I'm trying to make a level, but it keeps giving me an error.
I think the code itself is too old, and you need to re-create this with newer versions of c++ and h files.

**Comment by RIOT-Panko** [comment]
> I'm trying to make a level, but it keeps giving me an error.
> I think the code itself is too old, and you need to re-create this with newer versions of c++ and h files.

well that makes sense but i don't know c++ :c

**Comment by IPlayMinecraftYT** [comment]
> well that makes sense but i don't know c++ :c
I do know c++ and it doesn't make a lot of sense to me either. look up some tutorials online or somthing, just don't be lazy and give up like me, also read the readme.txt thoroughly

**Comment by RIOT-Panko** [comment]
> > well that makes sense but i don't know c++ :c
> > I do know c++ and it doesn't make a lot of sense to me either. look up some tutorials online or somthing, just don't be lazy and give up like me, also read the readme.txt thoroughly

what is the error that engine gives to you ?

**Comment by ehx-v1** [comment]
> I do know c++ and it doesn't make a lot of sense to me either. look up some tutorials online or somthing, just don't be lazy and give up like me, also read the readme.txt thoroughly

Yeah, legacy code is always "fun" to deal with. I might contribute a cleanup eventually, not promising anything though.

**Comment by bluefield-creator** [comment]
This is unmaintained
